internal/rtpbuffer/rtpbuffer.go

```diff
@@ -67,6 +67,7 @@ func (r *RTPBuffer) Add(packet *RetainablePacket) {
 			}
 			r.packets[idx] = nil
 		}
+		r.lastAdded = seq
```

Comment:
maybe rename to something like `newest` or `highestAdded` or something like that.

Reply:
Done